documents/order.md

```diff
@@ -272,7 +272,82 @@ client.order.edit(orderId,{
 }
 ```
 -------------------------------------------------------------------------------------------------------
+### View RTO/Risk Reasons
 
+```py
+orderId = "order_DaaS6LOUAASb7Y";
+
+client.order.viewRtoReview(orderId)
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
+| orderId* | string | The unique identifier of an order to access the fulfillment information. |
+
+**Response:**
+```json
+{
+  "risk_tier": "high",
+  "rto_reasons": [
+    {
+      "reason": "short_shipping_address",
+      "description": "Short shipping address",
+      "bucket": "address"
+    },
+    {
+      "reason": "address_pincode_state_mismatch",
+      "description": "Incorrect pincode state entered",
+      "bucket": "address"
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Update the Fulfillment Details
+
+```py
+orderId = "order_DaaS6LOUAASb7Y";
+
+request = {
+  "payment_method": "upi",
+  "shipping": {
+      "waybill": "123456789",
+      "status": "rto",
+      "provider": "Bluedart"
+  }
+}
+
+client.order.editFulfillment(orderId, request)
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
+| orderId* | string | The unique identifier of an order to access the fulfillment information. |
+| payment_method | string | The id of the order to be retrieve payment info |
```

Comment:
Is the `payment_method` description a proper one?

Reply:
added all possible values for payment_method